lib/csv_row_model/export/attributes.rb

```diff
@@ -0,0 +1,17 @@
+module CsvRowModel
+  module Export
+    module Attributes
+      extend ActiveSupport::Concern
+
+      class_methods do
+        # Safe to override. Method applied to each cell by default
+        #
+        # @param cell [Object] the cell's value
+        def format_cell(cell)
```

Comment:
let's change the method signature to be `format_cell(cell, column_name, column_index)` to be consistent... and just incase we want logic relating the `column_name/column_index`